lib/index.js

```diff
@@ -10,6 +10,8 @@ const pathExists = require('path-exists')
 const semver = require('semver')
 const sumchecker = require('sumchecker')
 
+let tmpFileCounter = 0
```

Comment:
@develar That syntax isn't supported but the syntax `ClassName.tmpFileCounter = 0` does work and has the same affect 👍